calour/dsfdr.py

```diff
@@ -95,7 +95,7 @@ def spearman(data, labels):
 
 
 # new fdr method
-def dsfdr(data, labels, method='meandiff', transform='rankdata', alpha=0.1, numperm=1000, fdrmethod='dsfdr'):
+def dsfdr(data,labels, transformtype='rankdata', method='meandiff', alpha=0.1, numperm=1000,fdrmethod='dsfdr'):
```

Comment:
pep8 likes spaces after comma (the one after data, is missing. should be data, labels and not data,labels)
also maybe name transform_type instead of transformtype